README.md

```diff
@@ -1,30 +1,55 @@
 # A guidelines and policy template for the development of research software at German research institutions
 ## A recommendation by GI, RSE working group and de-RSE
 
-Research Software Engineering (abbreviated: RSE) is the application of software engineering practices to research software, i.e., software created for and primarily used in scientific research projects. RSE is not High-Performance Computing (HPC), Computational Science Engineering (CSE), Artificial Intelligence, or Data Science. It has different goals and a distinct professional profile for Research Software Engineers.
+Research Software Engineering (abbreviated: RSE) is the application of software engineering practices to research software,
+i.e., software created for and primarily used in scientific research projects.
+RSE is not High-Performance Computing (HPC), Computational Science Engineering (CSE), Artificial Intelligence, or Data Science.
+It has different goals and a distinct professional profile for Research Software Engineers.
 
 Due to the increased relevance and complexity of research software, efficient and high-quality software development for research software also requires a range of organizational, methodological, and legal measures.
-This results in consequences both for the leadership to provide organizational support and for the individuals involved in software development to adhere to certain methodological guidelines. These guidelines were created based on the distilled knowledge (as of 2024) about software development in general (Software Engineering) and research software in particular (Research Software Engineering). It is planned to regularly update these guidelines and incorporate new insights.
+This results in consequences both for the leadership to provide organizational support and for the individuals involved in software development to adhere to certain methodological guidelines.
+These guidelines were created based on the distilled knowledge (as of 2024) about software development in general (Software Engineering) and research software in particular (Research Software Engineering).
+It is planned to regularly update these guidelines and incorporate new insights.
 
 The authors propose these guidelines as a template to aid universities, colleges, and research institutions in the adaptation of fundamental policies and recommendations for research projects involving research software.
 
+
```

Comment:
I thought this is now completed and has been approved by the GI as well, so that we should state that this is the version 1.0.0?